setuptools_scm/version.py

```diff
@@ -145,6 +148,31 @@ def guess_next_dev_version(version):
         return guess_next_version(version.tag, version.distance)
 
 
+def guess_next_simple_semver(version, distance, retain, increment=True):
+    parts = str(version).split('.', retain)
+    del parts[retain:]
+    while len(parts) < retain:
+        parts.append("0")
+    if increment:
+        parts[-1] = str(int(parts[-1]) + 1)
```

Comment:
It feels a little awkward here to be working with strings as numbers. It seems it'd be preferable to me to parse everything as numbers, and then convert back to a string at the end. But that may not be possible if some of the parts are meant to be non-numbers.

Reply:
unfortunately its permissible for `parts[0]` to be `'v123'`